spring-boot/src/test/java/org/dcsa/conformance/frontend/SeleniumTestBase.java

```diff
@@ -108,45 +127,50 @@ void runScenarioGroups(String name) {
         log.info("Stopping after first scenario group");
         break;
       }
-      log.info("Finished scenario group {}.", i + 1);
     }
   }
 
   private boolean handleJsonPromptForText() {
```

Comment:
```suggestion
  private boolean handleJsonPromptText() {
```
Rename this one as well?